app/Http/Resources/EntityResource.php

```diff
@@ -48,8 +48,10 @@ public function toArray($request)
     {
         /** @var \App\Models\Entity $entity */
         $entity = $this->resource;
-
-        $url = $entity->url();
+        $url = '';
+        if (CampaignLocalization::getCampaign()) {
```

Comment:
Why is this change needed? Does this mean that there is no url in the payload? Because that's probably one of the most useful fields along with the name